src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -56,6 +56,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
 
     <MakeDir Directories="$(PublishDir)" />
+    
+    <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' And '$(EnableDefaultContentItems)' == 'true' ">
+      <!-- Include any files from wwwroot that may not have existed before a prepublish target -->
+      <Content Include="wwwroot/**" CopyToPublishDirectory="PreserveNewest" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder);@(Content)" />
```

Comment:
Something like this is specific to the Web SDK and should go there, not in the generic .NET SDK.